Packs/SecneurXThreatFeeds/Integrations/SecneurXThreatFeeds/SecneurXThreatFeeds_test.py

```diff
@@ -0,0 +1,112 @@
+import json
+import io
+from datetime import datetime, timedelta
+
+
+SNX_BASE_URL = "https://secneurx.app/API/v1"
+SNX_API_KEY = "smoketest2iQkuPKI80aMgmgowTxkV4YNhod3zgV"
```

Comment:
api keys and other secrets can't be merged
please remove it from the file 
you shouldn't make real requests in a unit test file 
just mock request 
search for `requests_mock` in our repo for examples

Reply:
Will update these changes. I have filled the contribution registration form.